### PR TITLE
docs: change the docs buttons to cards for all products.

### LINKED ISSUE #68: docs: design and implement better main navigation
The navigation of the doc site is implemented using simple buttons which makes hard adding of additional links to the API site and etc.  

image

For this reason, it would be valuable if the design is changed to use bigger boxes which could allow multiple links to be added inside.

### PR DESCRIPTION
**Pull request by IvelinYR**
fixes #68

Screenshot from 2020-02-07 12-50-58

Screenshot from 2020-02-07 12-51-16

screencapture-localhost-8000-bg-2020-02-07-13_11_45

### CODE REVIEW

File: src/pages/_index.js [hunk 1]
```diff
@@ -1,43 +1,80 @@
...
+                title: 'GPS Tracking System',
+                addition: 'Simple to use',
+                description: 'Fleerp is affordable GPS Tracking System, successfully used by many companies, public sectors and personal households all around the world.'
+            },
+            {
+                icon: iamIcon,
+                image: iamImage,
+                name: 'IAM',
+                url: '/iam/docs/get-started/user-registration',
+                title: 'Mobile and Web sales',
```
Comment (mgenov) [inline src/pages/_index.js]: Identity & Access Management

Comment (mgenov) [reply src/pages/_index.js]: Fine-grained access control and visibility for centrally managing app resources.

Comment (IvelinYR) [reply src/pages/_index.js]: Ready with changes.

**Comment by mgenov** [comment]
The build is failing due some reason.

**Comment by mgenov** [comment]
See Docs need to be translated too.

File: src/components/ProductCard/ProductCard.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                      bg-blue-800 hover:text-white hover:bg-blue-700'>
+                    {button}
+                </Link>
+            </div>
+        </div>
+    </div>
+)
+
+ProductCard.propTypes = {
+    // icon: PropTypes.any.isRequired,
```
Comment (mgenov) [inline src/components/ProductCard/ProductCard.js]: commented props

Comment (IvelinYR) [reply src/components/ProductCard/ProductCard.js]: done

**Review by mgenov — APPROVED** [review]
LGTM!

**Pull request merged**